Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -113,4 +113,19 @@ extension String {
                                               range: range,
                                               withTemplate: replacement)
     }
+
+    /// Updates string once using the first occurrence of a regex
+    /// - Parameters:
+    ///   - replacement: New string to replace the matched pattern
+    ///   - regex: The regex pattern that will be matched for replacement
+    internal mutating func updateOnceExpression(replacement: String, regex: NSRegularExpression) {
+        var range = regex.rangeOfFirstMatch(in: self, options: [], range: range)
```

Comment:
I don't think this will compile. `range` is needed as the 3rd argument to `rangeOfFirstMatch`